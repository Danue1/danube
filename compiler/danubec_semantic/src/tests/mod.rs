use crate::{Context, semantic};

#[test]
fn all_files() {
    let context = Context {
        path: std::path::PathBuf::from("src/tests/fixtures/lib.dnb"),
    };
    let (fs, table, symbols, diagnostic) = semantic(context);

    insta::assert_debug_snapshot!(fs);
    insta::assert_debug_snapshot!(table);
    insta::assert_debug_snapshot!(symbols);
    insta::assert_debug_snapshot!(diagnostic);
}
