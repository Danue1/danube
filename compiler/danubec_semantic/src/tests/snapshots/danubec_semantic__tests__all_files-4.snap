---
source: compiler/danubec_semantic/src/tests/mod.rs
expression: diagnostic
---
  [31m√ó[0m Use glob without a path
